### DISCUSSION TITLE
Sezioni

### DISCUSSION THREAD

**Discussion by manuelbarberi** [iisgiua/giuaschool#298] [Idee e proposte]
Si può lavorare all'idea di inserire la stessa sezione ma indirizzi diversi

**Comment by manuelbarberi** [comment]
è una modifica che bisognerebbe fare al livello di grafica... a livello di database è molto semplice... già provato a fare a livello di database

**Reply by scuolagioco** [reply]
scusi se mi intrometto.. ha detto che ha provato a livello di database, ma ha funzionato?

**Comment by manuelbarberi** [comment]
Si

Il giorno gio 14 lug 2022 alle 13:13 Scuola Gioco ***@***.***>
ha scritto:

**Reply by scuolagioco** [reply]
e la conclusione di ciò quale sarebbe? è cambiato qualcosa nel registro?

**Comment by manuelbarberi** [comment]
Se è possibile avere due 1 in due sedi diverse con la stessa sezione a
livello di area admin…. Utenti inesperti non sanno come modificare

Il giorno gio 14 lug 2022 alle 13:59 Scuola Gioco ***@***.***>
ha scritto:

**Reply by scuolagioco** [reply]
ottimo.. era quello che mi serviva. mi potrebbe dare una mano, perché dovrei iniziare a organizzare le classi per settembre

**Reply by manuelbarberi** [reply]
@scuolagioco certamente mi può contattare sulla mia mail giuaschoolregistro@gmail.com per qualsiasi situazione/problematica/aiuto inerente al registro.

**Comment by trinko** [comment]
Al momento il suggerimento di @manuelbarberi di utilizzare le SEDI scolastiche come indirizzi diversi, permette di risolvere il problema (ma richiede una modifica sul database):

- SEDE 1:  Nome Scuola - Tecnico Informatico
  - classe 1A,2A 
- SEDE 2:  Nome Scuola - Liceo Scienze Applicate
  - classe 1A,2A 

In futuro conto di cambiare la gestione delle sezioni per consentire nomi lunghi (AInfo o simili).

**Comment by manuelbarberi** [comment]
Mi scriva al numero 3459746446

Il giorno gio 14 lug 2022 alle 14:06 Scuola Gioco ***@***.***>
ha scritto:
